webapp/channels/src/utils/performance_telemetry/index.ts

```diff
@@ -40,8 +43,8 @@ export function measureAndReport({
     canFail = false,
 }: {
     name: string;
-    startMark: string;
-    endMark?: string;
+    startMark?: string | DOMHighResTimeStamp;
```

Comment:
Could you leave `startMark` as required? It's technically not needed, but I'd rather keep it as required until we know we actually needed it